nodejs-example/README.md

```diff
@@ -1,23 +1,40 @@
-# Kotlin/Wasm Node.js Example
+# Kotlin/Wasm Node.js example
 
-A simple app printing "Hello World" in the Node.js.
+This example shows a simple application printing "Hello World" in Node.js.
 
-> **Note**
-> It uses a nightly version of Node.js 20
+![](screenshots/hello-world.png)
+
+> **Note:**
+> This application uses a nightly version of Node.js 20.
+
+In the next sections, you can find information to try out this application built with Kotlin/Wasm.
+
+## Set up the environment
+
+Before you start, prepare the IDE you require to run the application.
+
+### IDE
+
+We recommend using [IntelliJ IDEA 2023.1 or later](https://www.jetbrains.com/idea/) to work with the project.
+It supports Kotlin/Wasm out of the box.
 
 ## Build and run
 
-Check out the repo, navigate to the project folder, and use the following command:
+To build and run the application:
 
-### Run via Gradle
+1. In IntelliJ IDEA, open the repository.
+2. Navigate to the `nodejs-example` project folder.
+3. Run the application by using the following Gradle command:
 
 `./gradlew wasmJsNodeRun`
```

Comment:
I think we could add this command in step 3

Reply:
I tried it, but it is not that visible and hard to scan, I'll leave it as it is in a separate line.